### PR TITLE
Fix syntax errors in "Sending a push notification with error handling"

### PR DESCRIPTION
**Pull request by d1str0**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,34 +48,34 @@ c.Send(m)
...
-
-m := apns.NewNotification()
-m.Payload = p
-m.DeviceToken = "A_DEVICE_TOKEN"
-m.Priority = apns.PriorityImmediate
-m.Identifier = 12312, // Integer for APNS
-m.ID = "user_id:timestamp", // ID not sent to Apple – to identify error notifications
-
-c.Send(m)
+    c, err := apns.NewClientWithFiles(apns.ProductionGateway, cert.pem, key.pem)
```
Comment (nathany) [inline README.md]: I think you need quotes here for "cert.pem"

**Comment by nathany** [comment]
FYI, I would recommend looking into [Buford](https://github.com/RobotsAndPencils/buford) or [apns2](https://github.com/sideshow/apns2) to use Apple's modern HTTP/2 API. Error reporting is much better in the new API.

File: README.md (hunk 1 above)
Comment (d1str0) [inline README.md]: Yup. Found that out when I tried running it today. :P

**Comment by d1str0** [comment]
Thanks for those other suggestions. This was for a one off test, but I appreciate the info since I will probably need it later.

**Pull request merged**